### ISSUE TITLE
Unable to install rasa_core_sdk 0.14 with Rasa 0.14 

### ISSUE THREAD

**Issue by Lucioric2000** [RasaHQ/rasa#3377]
**Rasa version**: 0.14.1

**Python version**: 3.6.7

**Operating system** (windows, osx, ...): CentOS 7.5

**Issue**: When I installed the rasa.core version 0.14.1 with pip, (upgrading from rasa.core 0.13.7) it didn't allow me to update rasa_core_sdk  from 0.13 to 0.14, because it told that rasa-core 0.14.1 needed rasa_core~=0.13. But there appears a warning specifying that the major and minor version should be the same in rasa_core_sdk than in rasa_core for it to work better. looking in the repository, it appears tha the requirements.txt pins the version of rasa_core_sdk to 0.13. Please check this situation.
Best regards.

**Content of configuration file (config.yml)**:
```pipeline:
- name: WhitespaceTokenizer
- name: CRFEntityExtractor
# - name: ner_synonyms
- name: CountVectorsFeaturizer
- name: RegexFeaturizer
  token_pattern: (?u)\b\w+\b
- name: EmbeddingIntentClassifier
  epochs: 300
#- name: 'ner_duckling_http'
#  url: 'http://localhost:8000'
#  locale: "es_MX"
#  dimensions: ['time','PhoneNumber']
#  timezone: 'America/Mexico_City'
language: en
```

**Content of domain file (domain.yml)** (if used & relevant):
```yaml

```

**Comment by akelad** [comment]
Thanks for raising this issue, @tmbo  will get back to you about it soon.

**Comment by tmbo** [comment]
@Lucioric2000 yes you are right, there is an issue with that dependency. I have updated the requirement and an updated release is currently building on travis. the issue will be fixed with 0.14.2
